packages/fabric8-ui/src/app/home/link-error/link-error.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="blank-slate-pf col-sm-8 col-sm-offset-2" id="getting-started-error">
```

Comment:
The ID `getting-started-error` is the same as on `getting-started.component.html`. We only use IDs as a reference from the tests. Having duplicate IDs could cause an issue for the tests. I don't see that you reference the ID. If you don't need it please remove it. If this page is meant to duplicate `getting-started.component.html` then why not point to it. On that note, why do you have a duplicate of `getting-started.component.html`?

Reply:
@joshuawilson `getting-started`  component will be removed completely.